lib/js/components/Banner/Banner.vue

```diff
@@ -189,24 +146,45 @@
 		margin-top: $space-xxxxs;
 	}
 
-	&__text {
+	&__textWrapper {
+		flex: 1;
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+	}
+
+	&__titleWrapper {
 		display: flex;
 		flex-direction: column;
 		justify-content: center;
 		flex-grow: 1;
 		padding: $space-xxxxs 0;
 	}
 
-	&__iconContainer {
-		@include iconContainerVerticalStyles;
+	&__rightWrapper {
```

Comment:
items in this container aren't aligned

Reply:
@tweetgeek still valid

Reply:
its like in figma, wrapper has correct paddings, alignment to top of this wrapper

Reply:
Because in Figma, there is only one example with component that has the same height as the Button. You can put some test or smaller component to see the problem in DS. It's possible that bug is also present in Figma

Reply:
I think that I don't understand what the problem is? Ewa said it was to be aligned to the top with specific paddings, just like in figma, and that's what it is made. Should we make it dependent on the size of the component putted in the slot?

Reply:
I'm not sure where Ewa said that, maybe she didn't analyze all possible cases?
See the screenshot when I put 15x15px square into this slot. For me it looks off
![Zaznaczenie_916](https://user-images.githubusercontent.com/2725894/193031759-9e71f379-4116-4297-8a49-cf2e5cc404e1.png)

Reply:
Ok i understand, I asked about it https://bethinkteam.slack.com/archives/C012R8RM3A4/p1664458749846619?thread_ts=1663751823.320519&cid=C012R8RM3A4

Reply:
Will be addressed in another PR.